volmaster/daemon.go

```diff
@@ -83,8 +83,8 @@ func (d *DaemonConfig) Daemon(debug bool, listen string) {
 
 	getRouter := map[string]func(http.ResponseWriter, *http.Request){
 		"/policy/{policy}":             d.handlePolicy,
-		"/list":                        d.handleList,
-		"/get/{policy}/{volume}":       d.handleGet,
+		"/volume/list":                 d.handleList,
```

Comment:
should we use `volumes` instead of `volume`, just to be consistent with `snapshots` below?